eservice/pdo/eservice/pdo_helper.py

```diff
@@ -100,8 +100,7 @@ def read_from_file(cls, basename, data_dir = None, txn_keys = None, block_store
 
         :param file_name:  string, name of the file
         :param search_path: list of strings, directories to search for the data file
-        :param txn_keys: Used to sign the register_enclave transaction. For Sawtooth,
-                         this is of type TransactionKeys, while for CCF, this is of type ServiceKeys
+        :param txn_keys: Used to sign the register_enclave transaction. This is of type ServiceKeys
         """
```

Comment:
use of ServiceKeys as txn keys for CCF was a wierd choice, cannot remember the reasons, now. May be now, given that we will remove secp256k1 and skfs, CCF can also use TransactionsKeys for txn_keys, including rewriting the TransactionKeys Class?

Reply:
> May be now, given that we will remove secp256k1 and skfs

We removed sawtooth and skfs, not secp256k1.

> CCF can also use TransactionsKeys for txn_keys, including rewriting the TransactionKeys Class?

It's orthogonal to this PR, and I don't have strong opinion on that.